### PR TITLE
IL Termini 8

### PR DESCRIPTION
**Pull request by valentinamignosa**
encoding text division, cleaning up the autogenerated text and checking the edition with Bivona and photograph

### CODE REVIEW

**Review by simonastoyanova — CHANGES_REQUESTED** [review]
@valentinamignosa ISic0193, line 1, the encoding of the reverse letter should be in `<hi rend="reversed">`.  
ISic0194, line 3, the number is outside the `num` and we need a space between the num and the gap.
ISic0195, line 2, a(nnis) needs expanding.
ISic0198, lines 4-5, if a word is complete, leave a space between that word (and its encoding) and the preceding/following gap. 
ISic0199, 200, 202, 215, 233, 234, 235, same about gaps.
ISic0207, line 3, encode the number at the end.
ISic0236, line 4, remove the space inside the `num`.

**Review by simonastoyanova — CHANGES_REQUESTED** [review]
@valentinamignosa same gap issue as the previous commit, ISic0239, line 3 leave a space between the gap and the supplied.
ISic0240, legible but uninterpretable letters we put inside `<orig>`. If we think line 1 is part of a name, we can encode that too. I'm putting this inscription in the document for discussion.
ISic0241 - same about using `orig` for those.

**Comment by valentinamignosa** [comment]
@simonastoyanova **I'm answering here starting from your comments just to make our discussion simpler tomorrow:**
- ISic0193, line 1, the encoding of the reverse letter should be in <hi rend="reversed"> **[we should discuss about it]**
- ISic0194, line 3, the number is outside the num and we need a space between the num and the gap **[done]**
- ISic0195, line 2, a(nnis) needs expanding **[done]**
- ISic0198, ISic0199, 200, 202, 215, 233, 234, 235, 239: if a word is complete, leave a space between that word (and its encoding) and the preceding/following gap **[done but not everywhere to be consistent with Bivona's edition]**
- ISic0207, line 3, encode the number at the end **[done]**
- ISic0236, line 4, remove the space inside the num **[I can't find the number in this inscription]**
- ISic0240, 0241: **Bivona doesn't suggest names. Nevertheless, I am not sure I would say that we have here uninterpretable letters... let's discuss tomorrow.**

File: inscriptions/ISic0192.xml [hunk 1]
```diff
@@ -170,44 +175,17 @@
...
-                    <expan>
-                        <abbr>P</abbr>
-                        <ex>ublio</ex>
-                    </expan>
-                    Lo
-                    <supplied reason="lost"/>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <lb n="3"/>XXXV
+                    <lb n="1"/>Ex <g type="interpunct">·</g> <expan><abbr>d</abbr><ex>ecreto</ex></expan> <g type="interpunct">·</g> <expan><abbr>d</abbr><ex>ecurionum</ex></expan>
+                    <lb n="2"/><expan><abbr>l</abbr><ex>ocus</ex></expan> <g type="interpunct">·</g> <expan><abbr>p</abbr><ex>ublice</ex></expan> <g type="interpunct">·</g> <expan><abbr>d</abbr><ex>atus</ex></expan> <expan><abbr>P</abbr><ex>ublio</ex></expan> Lo<supplied reason="lost"/><gap reason="lost" extent="unknown" unit="character"/>
```
Comment (JonPrag) [inline inscriptions/ISic0192.xml:181]: There seems to be a redundant supplied element here?

**Comment by JonPrag** [comment]
"ISic0198, ISic0199, 200, 202, 215, 233, 234, 235, 239: if a word is complete, leave a space between that word (and its encoding) and the preceding/following gap [done but not everywhere to be consistent with Bivona's edition]"
This seems an interesting point - my first reactions is that this should depend upon whether the word end is coincident with the break on the stone, but in fact it highlights the point that we're using white-space to mark word-breaks, which i realise is uibiquitous in editions, but is actually an ultimately tacit form of editorian intervention? Can't help but wonder if this is a problem.

File: inscriptions/ISic0194.xml [hunk 2]
```diff
@@ -171,48 +176,20 @@
...
+                    <lb n="4"/><expan><abbr>L</abbr><ex>ucius</ex></expan> <g type="interpunct">·</g> Mammius <g type="interpunct">·</g> V<gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="5"/>uxor<hi rend="tall">i</hi> <g type="interpunct">·</g> cas<unclear>t</unclear><supplied reason="lost">issimae</supplied>
                 </ab>
-                <!--This EDR text 127383 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app><note>Text of ILTermini with slight changes based on the photograph</note></app>
+                    <app><note>wedge-shaped interpunct according to Bivona and also clear in the photograph</note></app>
```
Comment (JonPrag) [inline inscriptions/ISic0194.xml:191]: part of 'hand-note' rather than apparatus?

**Pull request merged**